src/main/java/org/springframework/social/salesforce/api/Salesforce.java

```diff
@@ -39,6 +39,8 @@
 
     public UserOperations userOperations();
 
+    public UserOperations userOperations(String gatewayUrl);
```

Comment:
Right, my point was just that pattern is usually instantiate thing, set properties, call useful method on thing rather than passing the property to set in useful method.

Having the argument in the useful method might be misleading.  Is `userOperations(String gatewayUrl)` called anywhere other than `SalesforceAdapter`?